.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dots-patterns"]
+	path = dots-patterns
+	url = git@github.com:simplysecure/dots-patterns.git
```

Comment:
I notice this is failing on netlify, do we need to change this to http rather than SSH or would it make more sense to add the appropriate key to netlify?

Reply:
It's expected so far because I didn't push my branch to `dots-patterns`. It'll go away I hope :)